esp/src/eclwatch/TargetClustersQueryWidget.js

```diff
@@ -0,0 +1,323 @@
+            });
+            this.legacyTargetClustersIframeWidget.placeAt(this._tabContainer, "last");
+            this.machineFilter.disable();
+        },
+
+        createGrid: function (domID) {
+            var context = this;
+            var retVal = new declare([ESPUtil.Grid(true, true, false, true)])({
+                store: ESPPreflight.CreateTargetClusterStore(),
+                columns: {
+                    col1: selector({
+                        width: 20,
+                        selectorType: 'checkbox'
+                    }),
+                    Configuration: {
+                        label: this.i18n.Configuration,
+                        renderHeaderCell: function (node) {
+                            node.innerHTML = Utility.getImageHTML("configuration.png", context.i18n.Configuration);
+                        },
+                        width: 10,
+                        sortable: false,
+                        formatter: function (configuration) {
+                            if (configuration === true) {
+                                return "<a href='#' />" + Utility.getImageHTML("configuration.png", context.i18n.Configuration) + "</a>";
+                            }
+                            return "";
+                        }
+                    },
+                    Name: tree({
+                        formatter: function (_name, row) {
+                            var img = "";
+                            var name = _name;
+                            if (row.type === "targetClusterComponent") {
+                                name = "<a href='#' class='dgrid-row-url'>" + row.Netaddress + " - " + _name + "</a>";
+                            }
+                            return img + "&nbsp;" + name;
+                        },
+                        expand: true,
+                        label: this.i18n.Name,
+                        collapseOnRefresh: false,
+                        sortable: true,
+                        width: 150,
+                        shouldExpand: function (clusterProcess) {
+                            if (clusterProcess.data.type === "targetClusterProcess") {
+                                return true;
+                            }
+                            return false;
+                        }
+                    }),
+                    Node: {
+                        label: this.i18n.Node,
+                        sortable: false,
+                        width: 100
+                    },
+                    Platform: {
+                        label: this.i18n.Platform,
+                        sortable: false,
+                        width: 75
+                    },
+                    Directory: {
+                        label: this.i18n.Directory,
+                        sortable: false,
+                        width: 200
+                    },
+                }
+            }, domID);
```

Comment:
Should 1 col have no width?

Reply:
When applying no width to a column in this grid it causes all sorts of column sizing issues.